src/components/TabBar/TabBar.jsx

```diff
@@ -47,14 +49,21 @@ const StyledSection = styled.section`
   }
 `;
 
+const filteredURLSearchParams = () => {
+  const PARAMS_TO_IGNORE = [PAGINATION_PAGE_NUMBER, PAGINATION_PAGE_SIZE];
+  const params = querystring.parse(window.location.search.substring(1));
```

Comment:
I think you can just do `delete params[PAGINATION_PAGE_NUMBER]` and `delete params[PAGINATION_PAGE_NUMBER]` here and skip the overhead of the array creation and .forEach function